### ISSUE TITLE
Full support of keyboard and mouse control like on PC

### ISSUE THREAD

**Issue by res87th** [xyzz/openmw-android#38]
I have keyboard and mouse connected to my android device what I want to play with like on PC. Is there any chance to see this function as an option in one of the next release?

**Comment by PaddleStroke** [comment]
I agree this would be awesome, specially when using android desktop mode like samsung Dex.

**Comment by darekdeo** [comment]
Seems like keyboard works fine, only the mouse tracking support is needed as clicking works fine too.
